runtime/strategies/rulesets.js

```diff
@@ -26,48 +26,49 @@ import {ResolveRecipe} from './resolve-recipe.js';
 
 export const Empty = new Ruleset.Builder().build();
 
-export const ExperimentalPhased = new Ruleset.Builder().order(
-  [
-    InitPopulation,
-    InitSearch
-  ],
-  SearchTokensToParticles,
-  [
-    MatchRecipeByVerb,
-    MatchParticleByVerb
-  ],
-  ConvertConstraintsToConnections,
-  GroupHandleConnections,
-  [
-    CreateHandles,
-    AddUseHandles,
-    AssignRemoteHandles,
-    CopyRemoteHandles,
-    AssignHandlesByTagAndType,
-    MatchFreeHandlesToConnections,
-    FallbackFate,
-  ],
-  MapSlots,
-  CreateDescriptionHandle,
-  ResolveRecipe
-).build();
+// Note: rulesets below are not used yet. Will uncomment on demand.
+// export const ExperimentalPhased = new Ruleset.Builder().order(
```

Comment:
why are you commenting these out?

Reply:
they are not used, and `CreateHandles` is part of them - so one way is to comment out `CreateHandles` everywhere. but i figured, that it might be more clear that rulesets aren't used yet, if they are commented out.